pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -33,6 +33,12 @@ By convention, `full` indicates that the permission supports all other action id
 
 ## <a name="admin-router"></a>Admin Router Permissions
 
+In each of these cases the principal is the client performing the HTTP
+request. The principal is authenticated using the DC/OS Authentication Token it
```

Comment:
I like this paragraph, but I wonder what we gain by using the term 'principal' here .

Am I right that the major goal is to describe how Admin Router associates the HTTP request with a DC/OS user (i.e. DC/OS uid), and then to provide an example of how it evaluates permission data? If that is the case, I have one proposal:

"Most HTTP requests received by Admin Router require authentication proof. After extracting the DC/OS `uid` associated with an incoming HTTP request (by validating and evaluating the DC/OS authentication token presented in the `Authorization` HTTP request header) Admin Router checks that this user has been granted the necessary privilege to perform the requested operation. For example, the DC/OS user identified by said `uid` must have `full` access to the protected resource `dcos:adminrouter:package` in order to be able to access the DC/OS package API through Admin Router."

(I see we use "principal" here: https://docs.mesosphere.com/1.10/security/ent/perms-management/ -- but I think that is more or less an accident, and there we should also introduce "DC/OS user <uid>" or "DC/OS group <gid>".)

Reply:
That's a beautiful paragraph. I like `principal` because it is a pretty clear term whereas DC/OS user is not. Adam's comment shows how confused all this sounds: 

> "DC/OS user" rather than "end user"

I think there is a distinction between an IAM user account, and someone sitting in from of a computer accessing a DC/OS cluster. In my mind, `DC/OS user` refers most closely to an entry in the `users` table of the IAM. It is short for `DC/OS IAM regular user`. That includes service account users. I tried with 'end user' to imply a physical meaty creature sitting at a computer. Oh my word this is tough ><. 

I'm going to try and get rid of `principal` and just be clear that I mean `DC/OS user`.

Reply:
> Am I right that the major goal is to describe how Admin Router associates the HTTP request with a DC/OS user (i.e. DC/OS uid), and then to provide an example of how it evaluates permission data?

@jgehrcke that, too. My main goal of this PR was to make it clear that when Alice attemps an action which fails, it may very well be because a necessary permission has not been assigned to the `dcos_marathon` or other user and that assigning it to the `alice` won't help. In light of that I wanted it to be clear who the entity being authorized is in each case.

Reply:
I was referring to the goal of this paragraph. :-) 

> I like principal because it is a pretty clear term whereas DC/OS user is not. 

Maybe in this documention not yet. In our concepts it is. Let's change this.

In our concept/architecture there are DC/OS user accounts. Each user account is identified by a corresponding cluster-unique `uid`. There are 'regular user accounts' and 'service user accounts'.